.env

```diff
@@ -0,0 +1,5 @@
+DATABASE_URL="mysql://root:1234@127.0.0.1:3307/foundry_test?serverVersion=5.7.42"
+#DATABASE_URL="postgresql://zenstruck:zenstruck@127.0.0.1:5433/zenstruck_foundry?serverVersion=15"
```

Comment:
```suggestion
```

Reply:
shouldn't we keep it? it shows how to switch to a postgres database

Reply:
Ok, I've added an example in the README

Reply:
> shouldn't we keep it? it shows how to switch to a postgres database

Oh, my bad, then we should make it the dsn that works out of the box with the docker image.

Reply:
ok I'll do this

Reply:
actually I've added in the README how to switch to postgres, so I've removed the comment here